src/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: AvenirB;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-weight: 300;
+}
+
+.form__big {
+  display: block;
+  position: relative;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.input__big {
+  position: relative;
+  box-sizing: border-box;
+  height: 70px;
+  padding: 25px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+}
```

Comment:
there's a bunch of css-rules that are repeated in input__big and input__small.
Better to create separate rule for just `.input` where you add everything reusable.

Reply:
I had to git clone this again (made backup) then tried to push it back but could not because already exist (i guess?).
May I have some assist in this matter please?

Reply:
If you need some help with this issue, maybe I can help. Just message me in slack 👋🏻